ui/components/component-library/tag/tag.types.ts

```diff
@@ -17,6 +18,14 @@ export interface TagStyleUtilityProps extends StyleUtilityProps {
    * Additional classNames to be added to the Tag component
    */
   className?: string;
+  /**
+   * The name of the icon to be used in the Tag component
+   */
+  iconName?: IconName;
```

Comment:
why do we need both of these? should we not make the iconName the only param and then decide how it should be rendered in the component? Unless there is a use case for having a very big "custom" icon I think it would be cleaner to have all the icon props set by default and just pass in the name.

Reply:
Okay, good point. So are you suggesting having `iconName` be the only thing passed in for the Icon and not `iconProps` (set those by default)?